### PR TITLE
Zigbee examples update

### PR DESCRIPTION
**Pull request by P-R-O-C-H-Y**
## Description of Change
This PR fixes existing Zigbee examples as there were several changes in the `esp-zigbee-sdk` and also removes Zigbee trace which is not included in the build anymore.

### CODE REVIEW

**Review by lucasssvaz — APPROVED** [review]

**Review by me-no-dev — APPROVED** [review]

File: libraries/ESP32/examples/Zigbee/Zigbee_Light_Bulb/README.md [hunk 1]
```diff
@@ -39,6 +39,9 @@ You can do the following:
 * In the Arduino IDE go to the Tools menu and set `Erase All Flash Before Sketch Upload` to `Enabled`
 * In the sketch uncomment function `esp_zb_nvram_erase_at_start(true);` located in `esp_zb_task` function.
 
+By default, the coordinator network is open for 180s after rebooting or flashing new firmware. After that, the network is closed for adding new devices.
```
Comment (pedrominatel) [inline libraries/ESP32/examples/Zigbee/Zigbee_Light_Bulb/README.md:42]: flashing new the firmware...

File: libraries/ESP32/examples/Zigbee/Zigbee_Light_Bulb/README.md [hunk 2]
```diff
@@ -39,6 +39,9 @@ You can do the following:
...
+You can change by editing `esp_zb_bdb_open_network(180);` in `esp_zb_app_signal_handler` function.
```
Comment (pedrominatel) [inline libraries/ESP32/examples/Zigbee/Zigbee_Light_Bulb/README.md:43]: You can change it by...

File: libraries/ESP32/examples/Zigbee/Zigbee_Light_Switch/README.md [hunk 3]
```diff
@@ -39,6 +39,9 @@ You can do the following:
 * In the Arduino IDE go to the Tools menu and set `Erase All Flash Before Sketch Upload` to `Enabled`.
 * In the `Zigbee_Light_Bulb` example sketch uncomment function `esp_zb_nvram_erase_at_start(true);` located in `esp_zb_task` function.
 
+By default, the coordinator network is open for 180s after rebooting or flashing new firmware. After that, the network is closed for adding new devices.
```
Comment (pedrominatel) [inline libraries/ESP32/examples/Zigbee/Zigbee_Light_Switch/README.md:42]: Same here.

File: libraries/ESP32/examples/Zigbee/Zigbee_Light_Switch/README.md [hunk 4]
```diff
@@ -39,6 +39,9 @@ You can do the following:
...
+You can change by editing `esp_zb_bdb_open_network(180);` in `esp_zb_app_signal_handler` function.
```
Comment (pedrominatel) [inline libraries/ESP32/examples/Zigbee/Zigbee_Light_Switch/README.md:43]: And here.

File: libraries/ESP32/examples/Zigbee/Zigbee_Temperature_Sensor/README.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
+# Arduino-ESP32 Zigbee Temperature Sensor Example
+
+This example shows how to configure the Zigbee end device and use it as a HA temperature sensor.
```
Comment (pedrominatel) [inline libraries/ESP32/examples/Zigbee/Zigbee_Temperature_Sensor/README.md:3]: I think we can change "HA" to "Home Automation (HA)" for the first mention only.

File: libraries/ESP32/examples/Zigbee/Zigbee_Temperature_Sensor/README.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+ * This board means the board (e.g. ESP32-H2) loaded with `Zigbee_Temperature_Sensor` example.
+ * The remote board means the board (e.g. ESP32-H2) loaded with `Zigbee_Thermostat` example.
+
+Functions:
+ * After this board first starts up, it would be configured locally to report the temperature on 1 degree change and no periodic reporting to the remote board.
+ * By clicking the switch button (BOOT) on this board, this board will immediately send a report of the current measured temperature to the remote board.
+
+## Hardware Required
+
+* One development board (ESP32-H2 or ESP32-C6) acting as Zigbee coordinator (loaded with `Zigbee_Thermostat` example)
```
Comment (pedrominatel) [inline libraries/ESP32/examples/Zigbee/Zigbee_Temperature_Sensor/README.md:26]: Can you add the link to the example?

File: libraries/ESP32/examples/Zigbee/Zigbee_Temperature_Sensor/README.md [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+* A USB cable for power supply and programming
+* Choose another board (ESP32-H2 or ESP32-C6) as Zigbee end device (loaded with `Zigbee_Temperature_Sensor` example)
+
+### Configure the Project
+
+In this example, the temperature sensor task is reading the chip temperature.
```
Comment (pedrominatel) [inline libraries/ESP32/examples/Zigbee/Zigbee_Temperature_Sensor/README.md:32]: ...the internal temperature sensor...

File: libraries/ESP32/examples/Zigbee/Zigbee_Temperature_Sensor/README.md [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+
+## Troubleshooting
+
+If the End device flashed with this example is not connecting to the coordinator, erase the flash before flashing it to the board. It is recommended to do this if you did changes to the coordinator.
+You can do the following:
+
+* In the Arduino IDE go to the Tools menu and set `Erase All Flash Before Sketch Upload` to `Enabled`
+* In the sketch uncomment function `esp_zb_nvram_erase_at_start(true);` located in `esp_zb_task` function.
+
+By default, the coordinator network is open for 180s after rebooting or flashing new firmware. After that, the network is closed for adding new devices.
```
Comment (pedrominatel) [inline libraries/ESP32/examples/Zigbee/Zigbee_Temperature_Sensor/README.md:52]: ..the..

File: libraries/ESP32/examples/Zigbee/Zigbee_Temperature_Sensor/README.md [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+You can change by editing `esp_zb_bdb_open_network(180);` in `esp_zb_app_signal_handler` function.
```
Comment (pedrominatel) [inline libraries/ESP32/examples/Zigbee/Zigbee_Temperature_Sensor/README.md:53]: ..it..

File: libraries/ESP32/examples/Zigbee/Zigbee_Thermostat/README.md [hunk 10]
```diff
@@ -0,0 +1,82 @@
+# Arduino-ESP32 Zigbee Thermostat Example
```
Comment (pedrominatel) [inline libraries/ESP32/examples/Zigbee/Zigbee_Thermostat/README.md:1]: Please look at the changes from the other example and apply them to this one.

**Review by pedrominatel — COMMENTED** [review]
Thank you @P-R-O-C-H-Y for taking care of it.
I left some comments (minor changes) on the documentation/readme files.

File: libraries/ESP32/examples/Zigbee/Zigbee_Light_Bulb/README.md (hunk 1 above)
Comment (me-no-dev) [reply libraries/ESP32/examples/Zigbee/Zigbee_Light_Bulb/README.md:42]: new the what? 😄 The sentence looks good to me as-is

File: libraries/ESP32/examples/Zigbee/Zigbee_Light_Switch/README.md (hunk 3 above)
Comment (me-no-dev) [reply libraries/ESP32/examples/Zigbee/Zigbee_Light_Switch/README.md:42]: also this sentence is OK

File: libraries/ESP32/examples/Zigbee/Zigbee_Light_Switch/README.md (hunk 4 above)
Comment (me-no-dev) [reply libraries/ESP32/examples/Zigbee/Zigbee_Light_Switch/README.md:43]: change `it` by...

File: libraries/ESP32/examples/Zigbee/Zigbee_Temperature_Sensor/README.md (hunk 8 above)
Comment (me-no-dev) [reply libraries/ESP32/examples/Zigbee/Zigbee_Temperature_Sensor/README.md:52]: nope. it's ok

**Pull request merged**